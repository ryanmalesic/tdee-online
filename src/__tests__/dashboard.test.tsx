import React from 'react';
import renderer from 'react-test-renderer';

import Dashboard from '../pages/dashboard';

it('renders dashboard unchanged', () => {
  const tree = renderer.create(<Dashboard />).toJSON();
  expect(tree).toMatchSnapshot();
});
