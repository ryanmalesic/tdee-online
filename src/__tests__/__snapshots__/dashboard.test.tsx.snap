// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders dashboard unchanged 1`] = `
<section
  className="hero is-fluid is-fullheight background"
>
  <div
    className="hero-head heroHead"
  >
    <nav
      className="navbar"
      role="navigation"
    >
      <div
        className="navbar-brand"
      >
        <a
          className="navbar-item"
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <h1
            className="title"
          >
            TDEE Online
          </h1>
        </a>
        <a
          className="navbar-burger burger "
          data-target="navbarBasicExample"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
          tabIndex={0}
        >
          <span
            aria-hidden="true"
          />
          <span
            aria-hidden="true"
          />
          <span
            aria-hidden="true"
          />
        </a>
      </div>
      <div
        className="navbar-menu"
        id="navbarBasicExample"
      >
        <div
          className="navbar-start"
        >
          <a
            className="navbar-item"
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Home
          </a>
        </div>
        <div
          className="navbar-end"
        >
          <div
            className="navbar-item"
          >
            <div
              className="buttons"
            >
              <a
                className="button has-background-primary-dark"
                href="/signup"
                onClick={[Function]}
                onMouseEnter={[Function]}
              >
                <strong>
                  Sign up
                </strong>
              </a>
              <a
                className="button has-text-dark has-background-grey-lighter"
                href="/signin"
                onClick={[Function]}
                onMouseEnter={[Function]}
              >
                Sign in
              </a>
            </div>
          </div>
        </div>
      </div>
    </nav>
  </div>
  <div
    className="hero-body"
  >
    <div
      className="container has-text-centered"
    >
      <h1
        className="title is-1"
      >
        Loading...
      </h1>
    </div>
  </div>
  <div
    className="hero-footer footer"
  >
    <span
      className="tag"
    >
      Photo by  
      <a
        href="https://unsplash.com/@brett_jordan?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
      >
        Brett Jordan
      </a>
       on 
      <a
        href="https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText"
      >
        Unsplash
      </a>
    </span>
  </div>
</section>
`;
